### ISSUE TITLE
When adding .torrent using auto-add feature (watcher) UI window will goes to foreground

### ISSUE THREAD

**Issue by theerror** [transmission-remote-gui/transgui#1148]
When adding .torrent using auto-add feature (watcher) UI window will pop-up. Is it possible to change this behaviour? 

I would like, if it just add torrent and TransGUI window do not go to foreground. Maybe notification is acceptable, but not showing the window. Is it possible?

**Comment by antekgla** [comment]
You OS is MacOSX right?
What window is open on auto-add? The main UI or the Add Torrent Form?

My goal at design this feature was NO WINDOW is showed when TransGUI auto-add a torrent.
In Windows works that way...only a notificacion is showed.

Sorry I cant help you in this... I dont have any access to a Mac machine.

**Comment by theerror** [comment]
Yes, OSX. I do have disabled "prompt for download option" so only app will goes to foreground where appears new added torrents and it's starting of download. If I adding torrents in one by one and want to add, it's bit annoying.

**Comment by antekgla** [comment]
Programatically I dont do anything to bring the TransGUI UI to the foreground, so maybe this is a Mac problem....
If anyone with OSX and knowledge of Lazarus want to suggest anything...

**Comment by theerror** [comment]
Didn't you think about [VirtualBox with OSX](https://techsviewer.com/install-macos-high-sierra-virtualbox-windows/)?

 I can for sure test whatever, but understand, that's a bit complicated do it like this. Are there any logs or something?

**Comment by PeterDaveHello** [comment]
@theerror I suggest not to promote any hackintosh, illegal projects, or projects may have legal issues here ;)

**Comment by theerror** [comment]
@PeterDaveHello yep, you are right. Apple do not provide way how to test application for development purpose, as MS do? I know, that at least MS provides vbox image with Windows for such purpose. Actually I didn't realise, that Apple is such dick, that they aren't. No Mac device, no OSX developer? Really they thinking like this?

**Comment by PeterDaveHello** [comment]
@theerror I agree with you, and I am not against research purpose, just don't want anyone gets trouble with legal issues 😉

**Comment by xeverse** [comment]
Where's that 
watch-dir feature?

ui win will appear --
that's exactly the app 
behaviour i crave coz
i need to change the dest-dir
for almost each & every one torr..

i'll say -- the best torrent gui
4 liGnux i've ever seen -- at last..
& teh Last Active sort-mode 
is here; ffinally -- terrrific..
